src/index.js

```diff
@@ -133,7 +133,8 @@ module.exports = (
         ))
           return callback();
       } while (pkgPath.length > (pkgPath = pkgPath.substr(0, pkgPath.lastIndexOf(sep, pkgPath.length - 15 - request.length)) + sep + 'node_modules' + sep + request).length);
-      console.error(`ncc: Module directory "${context}" attempted to require "${request}" but could not be resolved, assuming external.`);
+      if (debugLog)
+        console.error(`ncc: Module directory "${context}" attempted to require "${request}" but could not be resolved, assuming external.`);
```

Comment:
I'm torn on this one. In production builds, its nice to know what was assumed as external. On the other hand, I don't think I have seen and bugs caused by the assumption of `external`

Reply:
Ok, if we're not sure lets leave it in for now.